src/inject.js

```diff
@@ -32,11 +32,13 @@ const injectStyleTag = (cssContents) => {
         }
     }
 
-    if (styleTag.styleSheet) {
-        styleTag.styleSheet.cssText += cssContents;
-    } else {
-        styleTag.appendChild(document.createTextNode(cssContents));
-    }
+    return styleTag;
```

Comment:
Based on what we have here, it seems like this function should take an array of styles and should either loop over them and call `styleTag.sheet.insert`, or `.join("")` them and fallback to the current code if `styleTag.sheet` doesn't exist.